news/980.feature

```diff
@@ -0,0 +1,2 @@
+Expand ``~`` prefix to user directory in path options, configs, and environment
+variables.
```

Comment:
If you don't plan supporting path/url options in this PR, it might be useful to enumerate those options where it's not supported.